Add 'tips for applying for teacher training' page

peteryates opened the pull request:
### Context and changes

This is a draft of the new 'tips on applying for teacher training' page.

The content was created by @rosecatt and @emmafrith.

peteryates committed:
Add 'tips for applying for teacher training' page

peteryates committed:
Add a link to the tips page from steps

peteryates committed:
Replace School Experience link placeholder

HettieS commented:
Just taking a look at this, not sure if it's just me but having the main heading, and then the two sentences of text looks a little strange to me. I think some of our other pages have the main heading, followed by the blue style heading underneath, and it looks a bit cleaner. What do you think?

![Screenshot 2021-04-21 at 15 33 27](https://user-images.githubusercontent.com/43522239/115571781-3a8d8480-a2b7-11eb-85e8-9e7c9e3535e8.png)

HettieS commented on content/tips-on-applying-for-teacher-training.md:
```diff
@@ -0,0 +1,180 @@
+
+You may want to ask potential training providers questions, such as:
+
+1. How are academic work and practical teaching arranged?
```
do we want this to be a numbered list? Or just bullet points? I don't think i've seen this style of numbering anywhere else on site

peteryates replied:
Good question. I don't think the order matters. Thoughts @EmmaFrith?

EmmaFrith replied:
I don't think questions lend themselves well to bullets - normally bullets are for v short, sharp bits of content with no capitalisation or punctuation - hence the numbered list.

EmmaFrith commented:
> Just taking a look at this, not sure if it's just me but having the main heading, and then the two sentences of text looks a little strange to me. I think some of our other pages have the main heading, followed by the blue style heading underneath, and it looks a bit cleaner. What do you think?
> 
> ![Screenshot 2021-04-21 at 15 33 27](https://user-images.githubusercontent.com/43522239/115571781-3a8d8480-a2b7-11eb-85e8-9e7c9e3535e8.png)

personally don't see an issue with this - we'd essentially be creating a heading for no reason

HettieS commented on content/tips-on-applying-for-teacher-training.md:
```diff
@@ -0,0 +1,180 @@
+
+## Choose your referees
+
+You need 2 references to apply.
```
should we spell the number 'two'? Not sure what govt best practice is

EmmaFrith replied:
I think we should use gov.uk style on GIT - which uses numerals for numbers except one, or where it makes more sense to write it out - see https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style

peteryates replied:
The GDS styleguide says:

> Write all other numbers in numerals (including 2 to 9) except where it’s part of a common expression like ‘one or two of them’ where numerals would look strange.

We're not sticking to it 100% everywhere here so I'm happy with either.

EmmaFrith replied:
i think going with gov style is our best bet because it's well established and user tested - it's what i've been using in all the recent editing i've been doing

peteryates committed:
Remove 'who and what to teach' line

The referenced page has been cancelled and the line doesn't really make
sense without the link.

Also promote 'If you're invited to interview' line to a header as per
the origial doc.

HettieS commented on content/tips-on-applying-for-teacher-training.md:
```diff
@@ -0,0 +1,180 @@
+
+## Your personal statement
+
+Explaining why you want to be a teacher is the crux of your application - so
```
Just a small point but I wonder if everyone will understand the word 'crux'? Is it worth simplifying to 'most important'?

EmmaFrith replied:
also not a fan of crux - feel free to edit that out @HettieS @peteryates

HettieS replied:
@peteryates what do you reckon? Should we replace 'the crux' with 'an essential part'?

HettieS commented on content/tips-on-applying-for-teacher-training.md:
```diff
@@ -0,0 +1,180 @@
+A personal statement is usually about 500 words. It’s your chance to show your
+motivation, commitment and teaching potential.
+
+* You may want to cover some of the following:
```
Don't think this first bullet point 'You may want to...' is meant to be a bullet, think it's the intro sentence to the bullets

peteryates replied:
Well spotted! My mistake when importing. Fixing now.

HettieS commented on content/tips-on-applying-for-teacher-training.md:
```diff
@@ -0,0 +1,180 @@
+* personal qualities that will make you a good teacher
+* your contribution to the life of the school outside the classroom – for
+  example, running extra-curricular activities and clubs
+* details any experience you have working with children and what you learnt
```
typo, think should be 'details of...'

EmmaFrith replied:
good spot!

HettieS commented on content/tips-on-applying-for-teacher-training.md:
```diff
@@ -0,0 +1,180 @@
+* your suitability to teach a subject or group
+* evidence of the knowledge you have in your subject
+
+Think about <mark>who and what to teach</mark> before you apply.
```
this marked text, 'who and what to teach', shows up on the site as highlighted yellow text - is that intended? Do we use that style elsewhere on site?

peteryates replied:
I've fixed that already, it was a link to a page that doesn't yet exist (and now won't) - the line has been removed.

The `<mark>` just makes it yellow so it's clear something needs to be done!

HettieS commented on content/tips-on-applying-for-teacher-training.md:
```diff
@@ -0,0 +1,180 @@
+
+Think about <mark>who and what to teach</mark> before you apply.
+
+If you’re invited to interview
```
This should be a heading

peteryates replied:
Just fixed that in 62903f145f46

peteryates committed:
Remove the asterisk from the list description

HettieS commented on content/tips-on-applying-for-teacher-training.md:
```diff
@@ -0,0 +1,180 @@
+
+Congratulations!
+
+Before you start your training provider will check whether you’re safe to work
```
there is a typo here, think it should be something like 'Before you start your training, the provider will check...'

EmmaFrith replied:
good spot! feel free to edit @HettieS @peteryates

HettieS commented on content/tips-on-applying-for-teacher-training.md:
```diff
@@ -0,0 +1,180 @@
+If you’re missing certain qualifications they may also ask you to do an
+equivalence test or take a ‘subject knowledge enhancement’ course.
+
+If you do not get a place
```
should be a title as well

peteryates committed:
Promote line from paragraph to title

peteryates committed:
Make the list entry make sense

peteryates committed:
Add 'the' to make the sentence read properly

peteryates committed:
Fix a broken link in steps

Markdown format link in HTML file doesn't work 😢

HettieS committed:
update crux word choice

HettieS commented on content/tips-on-applying-for-teacher-training.md:
```diff
@@ -0,0 +1,179 @@
+Some people do not get on the course they want because their application has
+not convinced the admissions team that they’re right for it.
+
+Talk to an experienced teacher for help preparing your application.
```
looking at Rose's original doc, I think this line is supposed to be a link to the adviser service

Commit:
add Tget an adviser link

peteryates committed:
Use the internal link for the TTA signup app

peteryates committed:
Add tta-link to choose your referees section

Had to amend the copy slightly here so it's obvious where the link will
take us.

HettieS commented on content/tips-on-applying-for-teacher-training.md:
```diff
@@ -0,0 +1,179 @@
+If you’re applying for a salaried course, one of your references must be from
+an employer.
+
+If you’re struggling, get free one-on-one support with choosing your referees.
```
here, 'get free one-on-one support...your referees.' should be the TTA service link

Commit:
adding keywords
